tests/data_context/test_data_context.py

```diff
@@ -996,13 +996,18 @@ def test_build_batch_kwargs(titanic_multibatch_data_context):
     assert batch_kwargs["partition_id"] == "Titanic_1911"
 
 
-def test_existing_local_data_docs_urls_returns_nothing_on_empty_project(tmp_path_factory):
-    empty_directory = str(tmp_path_factory.mktemp("hey_there"))
+def test_existing_local_data_docs_urls_returns_url_on_project_with_no_datasources_and_a_site_configured(tmp_path_factory):
+    """
+    This test ensures that a url will be returned for a default site even if a
+    datasource is not configured, and docs are not built.
+    """
+    empty_directory = str(tmp_path_factory.mktemp("another_empty_project"))
     DataContext.create(empty_directory)
     context = DataContext(os.path.join(empty_directory, DataContext.GE_DIR))
 
     obs = context.get_docs_sites_urls()
-    assert obs == []
+    assert len(obs) == 1
+    assert obs[0].endswith("great_expectations/uncommitted/data_docs/local_site/index.html")
```

Comment:
Does this mean it returns a url even if that site does not in fact exist? (i.e. it's never been built?)

Reply:
It does. Here is my rationale on that behavior change. This was built with local disk stores in mind and doesn't use store apis. Therefore unexpected things might happen if the docs are a remote blob store. If the user has a site configured, the url could be retrieved regardless of build status.

Reply:
Put another way, I could imagine this being used programmatically and if we need to augment this to only return **built and existing urls** we could add a parameter, or create another method like `get_existing_data_docs_urls`.